cloud/backend/cf-storage-cors/index.ts

```
@@ -0,0 +1,42 @@
+  "Access-Control-Allow-Origin": "*",
+  "Access-Control-Allow-Methods": "GET, PUT, HEAD",
+  "Access-Control-Allow-Headers": "*",
+};
+
+export default {
+  async fetch(request: Request, env: Env): Promise<Response> {
+    if (request.method === "OPTIONS") {
+      return new Response(null, { status: 200, headers: CORS_HEADERS });
+    }
+
+    const url = new URL(request.url);
```

ESLint

[error] Unexpected use of 'URL'.

(no-restricted-globals)